src/class-email-types.php

```diff
@@ -164,10 +173,10 @@ public function presave( $value, $current_value = [] ) {
 	/**
 	 * Sort the email types on save so that they are always alphabetical.
 	 *
-	 * @param array $new_value The value being saved.
-	 * @return array
+	 * @param array<array{uuid4: string, label: string, image: int, templates: array<int>, from_name: string, safe_rtb?: string, ad_tags?: array<array{tag_code: string}>, roadblock?: bool, key_values?: array<array{key: string, value: string}>}> $new_value The value being saved.
+	 * @return array<array{uuid4: string, label: string, image: int, templates: array<int>, from_name: string, safe_rtb?: string, ad_tags?: array<array{tag_code: string}>, roadblock?: bool, key_values?: array<array{key: string, value: string}>}>
```

Comment:
Not the prettiest thing in the world, is it? Exactly what I dislike about TS it's so verbose

Reply:
100%! it definitely clutters things up, but hopefully the good outweighs the bad 😝